Library/Homebrew/language/python.rb

```diff
@@ -132,7 +132,7 @@ def virtualenv_create(venv_root, python = "python", formula = self)
           next unless dep_site_packages.exist?
           "import site; site.addsitedir('#{dep_site_packages}')\n"
         end
-        if pth_contents.any?
+        unless pth_contents.empty?
```

Comment:
I see. I think I will append `.compact` in the expression above, which makes the core more readable.